Database-basic/ER-diagram.drawio.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
![image](https://github.com/user-attachments/assets/0a8af8ce-b80a-4cb7-b3f9-c98e615d3abc)
![image](https://github.com/user-attachments/assets/6577c086-8088-4caf-a69c-050b022ed757)
One book cannot be ordered **two** times, so it seems that we do not need _quantity_ column in the `ordered_books` table